opentelemetry-ruby.gemspec

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'opentelemetry/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'opentelemetry-ruby'
+  spec.version       = OpenTelemetry::VERSION
+  spec.authors       = ['Ian Quick', 'Francis Bogsyani']
+  spec.email         = ['ian.quick@shopify.com']
+
+  spec.summary       = 'OpenTelemetry client library'
+  spec.homepage      = 'https://opentelemetry.com'
+  spec.license       = 'Apache'
+
+  spec.files         = Dir.chdir(File.expand_path('.', __dir__)) do
+    `git ls-files -z`.split("\x0").reject do |f|
+      f.match(%r{^(test|spec|features)/})
+    end
+  end
+  spec.require_paths = ['lib']
+  spec.required_ruby_version = '> 2.3.0'
```

Comment:
`>= 2.5.0`

Reply:
won't be able use mutexes then lolololol